rna/models.py

```diff
@@ -44,6 +56,7 @@ class Release(TimeStampedModel):
     bug_list = models.TextField(blank=True)
     bug_search_url = models.CharField(max_length=2000, blank=True)
     system_requirements = models.TextField(blank=True)
+    system_requirements_id = models.ForeignKey(SystemRequirement, blank=False, on_delete=models.CASCADE)
```

Comment:
But whatever route you take you still need to add database migrations to this PR. If you'd rather not do the dual field method, you could do it all with db migrations:

1. Add the new system requirements model and add a migration to create the db table
2. Add a new field for the foreign key to the release table with a temporary name
3. Create a data migration to copy all of the system requirement data to new entries in the new table and associate them with the release via the temporary field
4. Create a schema migration to delete the old system_requirements field from release
5. Create a schema migration to rename the temporary foreign key field to system_requirements

After that you'd have a row in the system requirements table for every release, but then the admins could begin consolidating if they wanted.